string name
uint32 red
uint32 green
uint32 blue
uint32 area
geometry_msgs/Point center
geometry_msgs/Point top_left
geometry_msgs/Point bottom_right